### PR TITLE
Add changelog for v1.16.8

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.16.8](https://github.com/go-gitea/gitea/releases/tag/v1.16.8) - 2022-05-16
+
+* ENHANCEMENTS
+  * [doctor] Add check/fix for bogus action rows (#19656) (#19669)
```
Comment (Gusted) [inline CHANGELOG.md]:
```suggestion
  * Add doctor check/fix for bogus action rows (#19656) (#19669)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
...
+  * Make .cs highlighting legible on dark themes. (Backport #19604) (#19605)
```
Comment (Gusted) [inline CHANGELOG.md]:
```suggestion
  * Make .cs highlighting legible on dark themes. (#19604) (#19605)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
...
+* BUGFIXES
+  * Fix oauth setting list bug (#19681)
+  * Delete user related oauth stuff on user deletion too (#19677) (#19680)
+  * Fix new release from tags list UI (#19670) (#19673)
+  * Prevent NPE when checking repo units if the user is nil (#19625) (#19630)
+  * GetFeeds must always discard actions with dangling repo_id (#19598) (#19629)
+  * Call MultipartForm.RemoveAll when request finishes (#19606) (#19607)
+  * Avoid MoreThanOne error when creating a branch whose name conflicts with other ref names (#19591)
```
Comment (Gusted) [inline CHANGELOG.md]:
```suggestion
  * Avoid MoreThanOne error when creating a branch whose name conflicts with other ref names (#19557) (#19591)
```

**Review by 6543 — APPROVED** [review]

**Review by Gusted — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
...
+  * Fix oauth setting list bug (#19681)
+  * Delete user related oauth stuff on user deletion too (#19677) (#19680)
+  * Fix new release from tags list UI (#19670) (#19673)
+  * Prevent NPE when checking repo units if the user is nil (#19625) (#19630)
+  * GetFeeds must always discard actions with dangling repo_id (#19598) (#19629)
+  * Call MultipartForm.RemoveAll when request finishes (#19606) (#19607)
+  * Avoid MoreThanOne error when creating a branch whose name conflicts with other ref names (#19557) (#19591)
+  * Fix sending empty notifications (#19589) (#19590)
+  * Ignore DNS error when doing migration allow/block check (#19566) (#19567)
+  * Add finalizers to ensure that repos are closed and blobreaders are closed (#19495) (#19496)
```
Comment (wxiaoguang) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,26 @@ This changelog goes through all the changes that have been made in each release
...
+* MISC
+  * Revert "Add finalizers to ensure that repos are closed and blobreaders are closed (#19495)" (#19659)
```
Comment (wxiaoguang) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,24 @@ This changelog goes through all the changes that have been made in each release
...
+  * Delete user related oauth stuff on user deletion too (#19677) (#19680)
+  * Fix new release from tags list UI (#19670) (#19673)
+  * Prevent NPE when checking repo units if the user is nil (#19625) (#19630)
+  * GetFeeds must always discard actions with dangling repo_id (#19598) (#19629)
+  * Call MultipartForm.RemoveAll when request finishes (#19606) (#19607)
+  * Avoid MoreThanOne error when creating a branch whose name conflicts with other ref names (#19557) (#19591)
+  * Fix sending empty notifications (#19589) (#19590)
+  * Ignore DNS error when doing migration allow/block check (#19566) (#19567)
+* MISC
+  * Fix issue overview for teams (#19652) (#19653)
```
Comment (wxiaoguang) [inline CHANGELOG.md]:
```suggestion
  * Fix issue overview for teams (#19652) (#19653)
```

The `MISC` section seems not necessary. The issue below is a bug fix.

**Comment by 6543** [comment]
:rocket:

**Comment by richmahn** [comment]
🎉

**Pull request merged**